packages/types/src/sdk/events/user.ts

```diff
@@ -4,20 +4,23 @@ export interface UserCreatedEvent extends BaseEvent {
   userId: string
   audited: {
     email: string
+    viaScim?: boolean
```

Comment:
We could probably pull this out of the `audited` block into the main body, this would be useful to have in posthog too. The `audited` fields are removed for privacy reasons.

Reply:
👍🏽

Reply:
I think the audited are used for creating the messages. Is there a reason on why we only use these `audited` fields? We could use the full event to populate the handlebars message, so we could move this new field into a higher level.
Is there any specific reason why it was not done like that? @mike12345567

Reply:
The `audited` props are simply there so that certain properties can be passed to the auditing system, but removed before sending to Posthog. Some private information like say `email` should not be passed up to Posthog as it is PII.

In the case of SCIM flag, it should be top level as we want to use this for auditing (the whole structure and the `audited` section) but also want to send it to Posthog (the structure minus the `audited` section).

Reply:
Ok. This means that `AuditedEventFriendlyName` will ingest the full event and the handlebars will use `user {{audited.email}}` instead of `user {{email}}`